katacomb/katacomb/continuum_pipeline.py

```diff
@@ -88,12 +86,11 @@ def decorator(builder):
     return decorator
 
 
-class Pipeline(object):
+class Pipeline(object, metaclass=ABCMeta):
```

Comment:
I think this gives the same effect:
```suggestion
class Pipeline(ABC):
```

Reply:
Yup. Fixed.